db/R/dbfiles.R

```diff
@@ -458,9 +466,15 @@ dbfile.check <- function(type, container.id, con, hostname=PEcAn.utils::fqdn(),
     
     if(nrow(dbfiles) > 1){
```

Comment:
Here and in the previous case, rather than having an additional ```if``` nested inside this iff, couldn't this line just be
```
if(nrow(dbfiles) > 1 &&  !return.all){
```